### PR TITLE
Refactor some provider classes

### PR DESCRIPTION
**Pull request by dkanada**
I decided to make a refactoring pull request before migrating the video providers to plugins.

### CODE REVIEW

File: MediaBrowser.Controller/Channels/IChannel.cs [hunk 1]
```diff
@@ -64,7 +64,7 @@ public interface IChannel
         /// </summary>
         /// <param name="type">The type.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns>Task{DynamicImageInfo}.</returns>
+        /// <returns>Task{DynamicImageResponse}.</returns>
```
Comment (Bond-009) [inline MediaBrowser.Controller/Channels/IChannel.cs]: NOTE: this should contain a description, not the type.

**Comment by dkanada** [comment]
None of them were in use anywhere as far as I can tell, and I don't think we use extras in the project, but I could be wrong.

**Comment by Bond-009** [comment]
Did you check plugins?

**Comment by dkanada** [comment]
Nothing in the organization uses them, so I doubt third party plugins are either.

File: MediaBrowser.Controller/Channels/IChannel.cs (hunk 1 above)
Comment (dkanada) [reply MediaBrowser.Controller/Channels/IChannel.cs]: Should I change that here or can I just leave it?

Comment (Bond-009) [reply MediaBrowser.Controller/Channels/IChannel.cs]: You can leave it for now

**Pull request merged**